src/DateUtils/DateUtils.js

```diff
@@ -1,29 +1,84 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
 
-import { format as formatDate, isMatch, parse as parseDate } from 'date-fns';
+import {
+  format as fnsFormat,
+  isMatch as fnsIsMatch,
+  isValid as fnsIsValid,
+  parse as fnsParse,
+  parseISO as fnsParseISO,
+} from 'date-fns';
+
+const decreaseDateByOffset = (date, offset) => {
+  if (date instanceof Date === false) {
+    console.error('decreaseDateByOffset parameter "date" must be a Date object');
+    return null;
+  }
+  return new Date(date - offset * 60 * 1000);
+};
+
+const increaseDateByOffset = (date, offset) => {
+  if (date instanceof Date === false) {
```

Comment:
As below you call the `decreaseDateByOffset` function that already check if _date_ is a Date object, is it really interesting to check it second time in this function ?